### DISCUSSION TITLE
关于页脚播放器的一点小问题

### DISCUSSION THREAD

**Discussion by xylu2024** [mirai-mamori/Sakurairo#407] [Q&A]
请问版主大大，APlayer的歌单列表点不开怎么解决呀，Aplayer的代码怎么定位在哪个位置呢，可能是我自己弄坏了，我可以自己鼓捣鼓捣

**Comment by KotoriK** [comment]
你可能在用旧版本
新版本的主题已经修复了歌单列表点不开的问题

**Reply by xylu2024** [reply]
感谢您的回复！网易云好多歌也听不了，我就又自己折腾着搞了个本地播放器哈哈哈哈
